counterpartylib/lib/messages/issuance.py

```diff
@@ -124,6 +137,36 @@ def validate (db, source, destination, asset, quantity, divisible, callable_, ca
         if destination:
             problems.append('cannot transfer a non‐existent asset')
 
+    # validate parent ownership for subasset
```

Comment:
In my opinion, a reissuance description should **not** contain the `;;lPARENT.child`.  There is no need to waste the additional bytes in the blockchain.

Related: `compose` will fail if the child asset is transferred and the new owner tries to compose an issuance specifying `PARENT.child` as the asset name.  Hmmm...  I think this should be fixed.

Are you ok with leaving out `;;lPARENT.child` from reissuance descriptions?  If so, then I should modify `compose` to handle a reissuance of a transferred child asset when with an asset name like `PARENT.child`.